src/pages/landing/MenuCard.tsx

```diff
@@ -13,7 +13,7 @@ import React from "react";
 export interface MenuCardProps {
     name: string;
     description?: string;
-    addAction?: () => void;
+    addAction?: (() => void) | null;
```

Comment:
For simplicity, instead of "null" we can return "undefined" when the conditions are false and keep the definition as it was before.

Reply:
Yep, it was my first approach as well but then I thought it was maybe cleaner to return a more explicit null. Anyway, will change it and merge. Thanks!